scripts/prepare-unity.sh

```diff
@@ -37,11 +37,28 @@ export GOPATH
 
 if [ -d "$IMPROBABLE_UNITY_VERSIONED" ]; then
   echo "Unity is already installed in:" $IMPROBABLE_UNITY_VERSIONED
-  exit 0
 else
   echo "Could not find a Unity $UNITY_PINNED_VERSION installation."
 fi
 
+if isWindows; then
```

Comment:
IMO these new additions can go within the `if [ -d "$IMPROBABLE_UNITY_VERSIONED" ]; then`